gen-api-reference/README.md

```diff
@@ -38,3 +38,40 @@ at the root of the repo, and that your GOPATH is set properly. For example:
 1. Run `git diff` to verify the changes.
 
 1. Create a PR to merge your changes.
+
+# Generating the Pipelines API Reference Page
+
+These are instructions for generating a copy of the [Pipelines API Reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/) page from the swagger file. 
+
+## Prerequisites
+* Download the [pipelines](https://github.com/kubeflow/pipelines) repository.
```

Comment:
- This document uses the git command "clone" to describe downloading a repository. I would recommend using it here as well. 
- I would recommend using the convention "organization/repository" when referring to a GitHub repository. In this case: kubeflow/pipelines

Reply:
Updated.